examples/rails.cson

```diff
@@ -0,0 +1,66 @@
+# This set of patterns will map common Ruby on Rails related files.
+
+# Test/specs for ruby files
+"^(.*[/\\\\])?(app|lib)/(.+)\\.rb$": [
```

Comment:
When we load the patterns in, we actually substitute / for whatever the current platform's path separator is, so we don't need to match both / and \.